src/components/icons/logo.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 
 const LogoIcon = ({ className }) => (
-  <svg className={className} viewBox="24 25 32 30">
+  <svg viewBox="24 25 32 30">
```

Comment:
```suggestion
  <svg className={className} viewBox="24 25 32 30">
```
This was just me trying to do something, wasn't meant to stay as a change.

Reply:
Reverted.